finagle-netty4/src/main/scala/com/twitter/finagle/netty4/channel/ServerBridge.scala

```diff
@@ -13,6 +13,8 @@ private[netty4] class ServerBridge[In, Out](
     serveTransport: Transport[In, Out] => Unit)
   extends ChannelInitializer[Channel] {
 
+  override def handlerAdded(ctx: ChannelHandlerContext): Unit = {}
```

Comment:
Yes, it is to prevent init to be called.
I also was considering to rebrand it, but after closer look at the sources of `ChannelInitializer` it seems like we will have to duplicate a lot of its code. So I decided the overriding `handlerAdded` is lesser of two evils.